documentation/api/io.strimzi.api.kafka.model.KafkaAuthorizationOpa.adoc

```diff
@@ -0,0 +1,44 @@
+To use link:https://www.openpolicyagent.org/[Open Policy Agent^] authorization, set the `type` property in the `authorization` section to the value `opa`.
```

Comment:
is the name of this adoc file like the FQDN of OPA authorization model purpose? it's `io.strimzi.api.kafka.model.KafkaAuthorizationOpa.adoc` which sounds weird to me.

Reply:
This is how the `@DescriptionFile` annotations works - when you set it on the class, it expects that there will be file named `<package>.<class-name>.adoc` which will be included into the API reference.